app/src/main/res/layout/view_donor_badge.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:parentTag="android.widget.FrameLayout">
+
+    <com.google.android.material.chip.Chip
+        android:id="@+id/donorChip"
+        style="@style/Chip.Accessible.Icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/contributions_dashboard_badge_donor"
+        android:textColor="?attr/paper_color"
+        app:chipBackgroundColor="?attr/progressive_color"
+        app:chipCornerRadius="16dp"
+        app:chipIcon="@drawable/ic_baseline_volunteer_activism_24"
+        app:chipIconTint="?attr/paper_color"
+        app:chipStrokeWidth="0dp"
+        app:rippleColor="?attr/overlay_color"
+        android:visibility="gone"
+        tools:text="Donor"
+        tools:visibility="visible" />
+
+    <com.google.android.material.chip.Chip
+        android:id="@+id/becomeADonorChip"
+        style="@style/Small"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/contributions_dashboard_badge_become_a_donor"
+        android:textColor="?attr/progressive_color"
+        app:chipBackgroundColor="?attr/paper_color"
+        app:chipCornerRadius="16dp"
+        app:chipIcon="@drawable/ic_baseline_volunteer_activism_24"
+        app:chipIconTint="?attr/progressive_color"
+        app:chipStrokeColor="?attr/border_color"
+        app:rippleColor="?attr/overlay_color"
+        android:visibility="gone"
+        tools:text="Become a donor"
+        tools:visibility="visible"/>
+
+    <com.google.android.material.chip.Chip
+        android:id="@+id/updateDonorStatusChip"
+        style="@style/Small"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/contributions_dashboard_badge_update"
+        android:textColor="?attr/progressive_color"
+        app:chipBackgroundColor="?attr/paper_color"
+        app:chipCornerRadius="16dp"
+        app:chipStrokeWidth="0dp"
+        app:rippleColor="?attr/overlay_color"
+        android:visibility="gone"
+        tools:text="Update donor status"
+        tools:visibility="visible"/>
```

Comment:
chips has default padding and since the background is transparent, the text will not align with the username in the article overflow menu. Removing the start padding looks something like this. 
1. [Without start padding](https://github.com/user-attachments/assets/114ebca7-546a-4422-aa86-ebc747c2e9cd)
2. [With default padding](https://github.com/user-attachments/assets/bcd76160-20b4-44de-af4f-350f9edd721f)

should we remove start padding and does this look good visually?

Reply:
Yes, you are correct! Changed it back to use `TextView`. Thanks!